doc/radosgw/notifications.rst

```diff
@@ -61,9 +61,15 @@ Asynchronous Notifications
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Notifications can be sent asynchronously. They are committed into persistent
-storage and then asynchronously sent to the topic's configured endpoint. In
-this case, the only latency added to the original operation is the latency
+storage and then asynchronously sent to the topic's configured endpoint.
+The notification will be committed into the persistent storage only if the triggering
+operation was successful.
+In this case, the only latency added to the original operation is the latency
 added when the notification is committed to persistent storage.
+If the endpoint of the topic to which the notification is sent is not available for a longer
+period of time, the persistent storage allocated for this topic will eventually fill up.
+Once this is happening we will start failing the triggering operations with "503 Service Unavailable" 
```

Comment:
When this happens the triggering operations will fail with `503 Service Unavailable`, which tells the client that it may retry later.